webstorage/event_constructor.html

```diff
@@ -1,32 +1,81 @@
 <!DOCTYPE HTML>
 <html>
  <head>
-  <title>WebStorage Test: StorageEvent - init value</title>
+  <title>WebStorage Test: StorageEvent - constructor</title>
```

Comment:
I've run these tests manually. They pass 5/5 in Chrome, Firefox in Safari. In Edge on real hardware and on BrowserStack however, the test seems to crash Edge, saying "This page is having a problem loading". @thejohnjansen FYI, there's probably a real bug here.

Reply:
I've narrowed it down to `new StorageEvent('storage', { url: null })`. I'll file a bug.

Reply:
I've filed https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19216299/ with this test case: http://software.hixie.ch/utilities/js/live-dom-viewer/saved/6254